src/NServiceBus.Testing.Tests/SagaTests.cs

```diff
@@ -245,6 +261,19 @@ protected override void ConfigureHowToFindSaga(SagaPropertyMapper<MySagaData> ma
         }
     }
 
+    public interface IDependency
+    {
+    }
+
+    public class MySagaWithConstructorDependency : MySaga
+    {
+
```

Comment:
We try to avoid unnecessary whitespaces. Could you remove those (I believe there should be a reformat configuration available if you're using R#)?